test/overlap.jl

```diff
@@ -63,3 +64,9 @@ end
 
 
 end
+
+@testset "position with respect to ellipse" begin
+    @test oblique_coefficients(2, 2, 0) == (0.25, 0.25, 0.0)
+    @test oblique_coefficients(2, 2, 90) == (0.25, 0.25, 0.0)
+    @test oblique_coefficients(2, 1, 30) == (0.4375, 0.8125000000000001, -6*sqrt(3)/16)
```

Comment:
It isn't a good idea to test exact equality ([`==`](https://docs.julialang.org/en/v1/base/math/#Base.:==)) of floating point numbers, because of their nature.

In these cases, you should rather test approximate equality with [`isapprox`](https://docs.julialang.org/en/v1/base/math/#Base.isapprox):
```suggestion
    @test all(oblique_coefficients(2, 1, 30) .≈ (0.4375, 0.8125, -6*sqrt(3)/16))
```
The same in the above lines